war/src/main/js/tooltips.js

```diff
@@ -9,12 +9,12 @@ const TOOLTIP_BASE = {
 
 let tooltipInstances = []
 const globalPlugin = {
-  fn() {
```

Comment:
htmlunit is being a big pain recently 😢

Reply:
It took a silly amount of time to find out what was causing it aha, not a fan :P 

--

Would you know what's causing this to happen?

<img width="825" alt="image" src="https://user-images.githubusercontent.com/43062514/163804716-fb35090b-d8c2-4c92-9ac2-cc700b83cebc.png">

Reply:
Agent termination, nothing to worry about, I retriggered the build.

Reply:
Wonderful - it's looking promising :)

Reply:
just swap to the ci.jenkins.io build or run `mvn deploy -Pquick-build` from the `war` directory.
For ATH anyway, bom needs incrementals

Reply:
> Agent termination, nothing to worry about

Well only if wasting time and money is nothing to worry about. I am worried about it, but addressing that problem is a topic for a separate thread.

Reply:
> > Agent termination, nothing to worry about
> 
> Well only if wasting time and money is nothing to worry about. I am worried about it, but addressing that problem is a topic for a separate thread.

The wording chosen seems to be misleading, I meant there's nothing to worry about in this PR, that is an issue for sure, but nothing that is caused by a change in this PR 👀

Reply:
Hopefully #6492 should fix incrementals publishing. And yes of course agent termination is not related to this PR. But at the same time, if we ignore the problem and never speak up about it, the problem will just continue to get worse and worse. Unfortunately providing constructive feedback to operators about such problems can be a challenge, as I discovered when I was on the receiving end of these complaints for several years as an operator myself. I am still trying to figure out what is going on here so that I can file a reasonable ticket for the Jenkins infrastructure team. Anyway, topic for another thread.